tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
@gmlewis Could be, I'm trying to remember the `goimports` logic. I thought it grouped it as stdlib first, then local packages second, then external last.